pinc/ProjectTransition.inc

```diff
@@ -314,7 +314,7 @@ class ProjectTransition
                 get_forum_id_for_project_state($this->to_state));
         }
 
-        if ($this->collateral_actions) {
+        if ($this->collateral_actions && !$quiet) {
```

Comment:
The collateral action might very well be something that is more than just echo'ing something to the screen. If you want to just ensure that nothing from the collateral function is printed to the screen, you could use `ob_start()` / `ob_end_clean()` to wrap the function to discard anything it would otherwise output.